engine/src/test/java/org/pentaho/di/job/entries/ping/JobEntryPingTest.java

```diff
@@ -69,41 +69,58 @@ protected Map<String, String> createSettersMap() {
   }
 
   private String[] valueList = {};
+  JobEntryPing jEPing = new JobEntryPing( "Ping" );
 
   @Test
   public void test_isNumeric_ForPositiveIntegers() {
-    valueList = new String[]{"22", "020", "   22   ", "2147483647"};
-    for (String value : valueList) {
-      assertTrue(JobEntryPing.isNumeric(value));
+    valueList = new String[]{ "22", "020", "   22   ", "2147483647" };
+    for ( String value : valueList ) {
+      assertTrue( JobEntryPing.isNumeric( value ) );
     }
   }
 
   @Test
   public void test_isNumeric_ForValueOtherThanPositiveIntegers() {
     //Integer.MAX_VALUE = 2147483648
-    valueList = new String[]{"-2", "0", "\n", "5.05", "", "abc", "ab412c", "2147483648", "-", "?"};
-    for (String value : valueList) {
-      assertFalse(JobEntryPing.isNumeric(value));
+    valueList = new String[]{ "-2", "0", "\n", "5.05", "", "abc", "ab412c", "2147483648", "-", "?" };
```

Comment:
"0" is a number and an integer by https://tutorme.com/blog/post/is-zero-an-integer. Since this varies from the standard definition. We should add a javadoc comment for the function `JobEntryPing#isNumeric` and explain that we aren't considering "0" as an integer. Or rename the function to `isPositiveNumber`

Reply:
Renaming the function seems to be a better idea. I will do that.